code/datums/mutations/sight.dm

```diff
@@ -39,10 +39,11 @@
 	name = "Thermal Vision"
 	desc = "The user of this genome can visually percieve the unique human thermal signature."
 	quality = POSITIVE
-	difficulty = 18
+	difficulty = 28
```

Comment:
this means you can't unlock it, as in, every single pair is random and not correct
please don't make random number changes

Reply:
This wasn't really a random number change.  It should be highly difficult to unlock, but not impossible.  At 18 people can still speedrun it in the first 10 minutes of a shift.  So, what would be the magic number to make it:

1) Not Impossible
2) Not speedrunable

Reply:
I moved it down to 24.  Which means if it's somehow -5 it'll be 19 missing sequences, otherwise it'll be 29 missing sequences.

Though I'm debating switching this to a gene recipe, not sure if that'd help prevent a speedrun or not.